torch/csrc/jit/operator.h

```diff
@@ -2,57 +2,69 @@
 // once C10 exists this can be removed, or stubbed out, but we need
 // it now to implement correct semantic checking for script
 #pragma once
-#include "ATen/ATen.h"
+
 #include "torch/csrc/jit/assertions.h"
 #include "torch/csrc/jit/ir.h"
 #include "torch/csrc/jit/function_schema.h"
 #include "torch/csrc/jit/stack.h"
 
+#include "ATen/ATen.h"
+
+#include <functional>
+#include <initializer_list>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
 namespace torch { namespace jit {
 
 FunctionSchema parseSchema(const std::string& schema);
 
 using OperationCreator = std::function<Operation(Node*)>;
 
 struct TORCH_API Operator {
-  Operator(FunctionSchema schema, OperationCreator op, OperationCreator op_const_attributes = nullptr)
-    : schema(std::move(schema))
-    , op(std::move(op))
-    , op_const_attributes(std::move(op_const_attributes)) {}
+  Operator(FunctionSchema schema, OperationCreator op_creator)
+      : schema(std::move(schema)), op_creator_(std::move(op_creator)) {}
 
-  Operator(const std::string& schema, OperationCreator op, OperationCreator op_const_attributes = nullptr)
-    : Operator(parseSchema(schema), std::move(op), std::move(op_const_attributes)) {}
+  Operator(const std::string& schema, OperationCreator op_creator)
+      : Operator(parseSchema(schema), std::move(op_creator)) {}
 
-  // Helper constructor to regsiter `op` to run
+  // Helper constructor to register `op` to run
   // run for _every_ IR Node where n.kind() == name, regardless of arguments.
-  // This is accomplished by marking the schema varargs and having no required arguments.
-  // This is used for things like prim::While or prim::If that can take a number
-  // of different valid input types and lengths.
-  Operator(Symbol name, OperationCreator op)
-  : Operator(FunctionSchema(name, {}, {}, true), op, op) {}
+  // This is accomplished by marking the schema varargs and having no required
```

Comment:
You should rebase on https://github.com/pytorch/pytorch/pull/9801 since it modifies Operator as well and will likely land today.